local/network.go

```diff
@@ -458,14 +461,22 @@ func (ln *localNetwork) Healthy(ctx context.Context) chan error {
 				// Every constants.HealthCheckInterval, query node for health status.
 				// Do this until ctx timeout
 				for {
+					// fast response if died previously
+					if !node.GetAliveStatus() {
+						return fmt.Errorf("unexpected stop on node %q", node.GetName())
```

Comment:
```suggestion
						return fmt.Errorf("node %q stopped", node.GetName())
```

Reply:
If the network is not stopped (in which case shall return earlier), then this node is stopped but was not requested to do that. That's why I use "unexpected stop" here and in other places.

Reply:
It's possible that we check whether the network is stopped, and it's not, and then we start the outer goroutine, and then the network is stopped. So actually, it's possible that the node isn't alive because the network was stopped.